admin/templates/common_auth/registration/invitation_email.html

```diff
@@ -0,0 +1 @@
+{% extends 'registration/password_reset_email.html' %}
```

Comment:
EOF.

Since some of these emails will be seen by more than just COS we should consider following the OSF in language used within the email.